frinx/workers/http_workers/http_workers.py

```diff
@@ -0,0 +1,83 @@
+from typing import Any
+
+from pydantic import AnyHttpUrl
+from pydantic import NonNegativeFloat
+
+from frinx.common.conductor_enums import TaskResultStatus
+from frinx.common.type_aliases import DictAny
+from frinx.common.type_aliases import DictStr
+from frinx.common.type_aliases import ListAny
+from frinx.common.type_aliases import ListStr
+from frinx.common.util import cookie_jar_to_dict
+from frinx.common.util import snake_to_camel_case
+from frinx.common.worker.service import ServiceWorkersImpl
+from frinx.common.worker.task_def import TaskDefinition
+from frinx.common.worker.task_def import TaskExecutionProperties
+from frinx.common.worker.task_def import TaskInput
+from frinx.common.worker.task_def import TaskOutput
+from frinx.common.worker.task_result import TaskResult
+from frinx.common.worker.worker import WorkerImpl
+from frinx.services.http_service import http_service
+from frinx.services.http_service.enums import ContentType
+from frinx.services.http_service.enums import HttpMethod
+from frinx.workers.http_workers.util import BasicAuth
+
+
+class HTTPWorkersService(ServiceWorkersImpl):
+
+    class GenericHTTPWorker(WorkerImpl):
+
+        class WorkerDefinition(TaskDefinition):
+            name: str = 'generic_HTTP_worker'
```

Comment:
use values from previous task definition             
"name": "HTTP_task",
"description": '{"description": "Generic http task", "labels": ["BASICS","HTTP"]}',

Reply:
OK, it's fixed.